services/web/client/source/class/osparc/dashboard/CardBase.js

```diff
@@ -434,6 +449,9 @@ qx.Class.define("osparc.dashboard.CardBase", {
       if (data.classifiers && data.classifiers.length) {
         return true;
       }
+      if ("folder" in data) {
+        return true;
```

Comment:
This shoukd never happen if I understand it correctly? When would `"folder" in data` be true but `this._filterFolder(data.folder)` be false?